src/dialects/abstract/query-interface.d.ts

```diff
@@ -214,6 +214,15 @@ export interface IndexesOptions {
 
 export interface QueryInterfaceIndexOptions extends IndexesOptions, QueryInterfaceOptions {}
 
+/**
+ * Interface for bulk delete options
+ */
+export interface QueryInterfaceBulkDeleteOptions {
+  truncate?: boolean
+  cascade?: boolean
+  restartIdentity?: boolean
```

Comment:
```suggestion
  truncate?: boolean;
  cascade?: boolean;
  restartIdentity?: boolean;
```

Reply:
Is this something we want to be picked up by eslint?

Reply:
Is it now that .d.ts is linted? If not, that's probably a bug with the eslint rule because it's part of our config